spec/access_policies/list_publication_group_access_policy_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe ListPublicationGroupAccessPolicy, type: :access_policy do
+  let(:anon)        { AnonymousUser.instance }
+  let(:user)        { FactoryGirl.create(:user) }
+  let(:admin)       { FactoryGirl.create(:user, :administrator) }
+  let(:application) { FactoryGirl.create(:doorkeeper_application) }
+
+  context '' do
+    xit '' do
+    end
+  end
```

Comment:
Not sure if I planned to add something here... maybe I'll just add the default shoulda_matchers stuff.